modules/arm_plugin/src/transformations/convert_interpolate_arm.cpp

```diff
@@ -98,6 +98,21 @@ ArmPlugin::pass::ConvertInterpolate::ConvertInterpolate() {
             return false;
         }
 
+        if (interp->get_input_size() == 4) {
```

Comment:
Why do we need to ensure the default axes order? As far as I could understand axes order is important only for input(1) and input(2) which aren't used anyway by ArmInterpolate since it rely on correct output(0) shape inference provided by IE core.

Reply:
Right, I removed this check